src/applications/vaos/appointment-list/components/UpcomingAppointmentsList.jsx

```diff
@@ -46,18 +41,14 @@ export default function UpcomingAppointmentsList() {
   useEffect(
     () => {
       if (futureStatus === FETCH_STATUS.notStarted) {
-        dispatch(
-          fetchFutureAppointments({
-            includeRequests: featureStatusImprovement,
-          }),
-        );
+        dispatch(fetchFutureAppointments({}));
```

Comment:
Remove empty brackets
```suggestion
        dispatch(fetchFutureAppointments());
```